conf/backup_method

```diff
@@ -59,15 +59,7 @@ This is an automated message from your beloved YunoHost server." | /usr/bin/mail
         exit $retcode
     fi
 
-    "$borg" prune --glob-archives "${name}-*" --list --keep-hourly 2 --keep-daily=7 --keep-weekly=8 --keep-monthly=12
-
-    # We prune potential manual backup older than 1 year
-    "$borg" prune --list --keep-within 1y
-
-    # Compact repo after pruning it, otherwise in fact nothing will be really removed from the repo,
-    # making its actual size grow disproportionately compared to what `borg info` displays
-    # See: https://github.com/borgbackup/borg/issues/6106#issuecomment-1215449050
-    "$borg" compact
+    "$borg" prune --glob-archives "${name}-*" --keep-hourly 2 --keep-daily=7 --keep-weekly=8 --keep-monthly=12
```

Comment:
Ah right, you seem to remove the `--list` argument here and above.
I am pretty OK with that, given the "noise" part I have raised in #239.

If no one object (otherwise I would like to argument in favor of that change too)